### PR TITLE
Cash register

### PR DESCRIPTION
**Pull request by ijassiem**

### CODE REVIEW

**Review by tyronevb — COMMENTED** [review]
Have a look at the proposed structure for commit messages. 

Out of interest, how are you entering your commit messages? Via `git commit -m "commit message"`? Or are you doing `git commit` which then launches a text editor for you to enter your commit message?

**Review by james-smith-za — COMMENTED** [review]
Put a file called `.gitignore` in the root of your git repo. There are quite a few files in your git repo that don't need to be here.

```
*.egg-info
__py_cache__
```

File: cash_register/cash_register.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    """Calculate total."""
+    total = 0
+    for ele in range(0, len(list)):
+        total = total + list[ele]
+    return round(total, 2)
+
+
+def vat(value):
+    """Calculate VAT."""
+    return round(value * 0.14, 2)
```
Comment (james-smith-za) [inline cash_register/cash_register.py]: VAT is now 15% ;-)

File: multiply.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"""
+This is an example docstring.
+
+A docstring with a summary line, a blank line
+and some more text like here.
+"""
```
Comment (james-smith-za) [inline multiply.py:6]: Make sure you update your docstrings. This will satisfy pydocstyle, but it's not terribly useful.

Comment (james-smith-za) [reply multiply.py:6]: Same for mod.py and divide.py

File: testing/test_cash_register.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+import cash_register.cash_register as cr
+import unittest
+
+"""
+This is an example docstring.
+
+A docstring with a summary line, a blank line
+and some more text like here.
+"""
```
Comment (james-smith-za) [inline testing/test_cash_register.py]: Same feedback here.

File: cash_register/cash_register.py (hunk 1 above)
Comment (ijassiem) [reply cash_register/cash_register.py]: Fixed

File: multiply.py (hunk 2 above)
Comment (ijassiem) [reply multiply.py:6]: These docstrings have been corrected in the functions branch.

File: testing/test_cash_register.py (hunk 3 above)
Comment (ijassiem) [reply testing/test_cash_register.py]: Fixed

**Pull request merged**